annotation/annotations.yaml

```
@@ -199,6 +199,14 @@ annotations:
     resources:
       - Pod
 
+  - name: sidecar.istio.io/statsHistogramBuckets
+    featureStatus: Alpha
+    description: Specifies the custom histogram buckets with a prefix matcher to separate the Istio mesh metrics from the Envoy stats, e.g. `{"istio":{1,5,10,50,100,500,1000,5000,10000},"envoy":{1,5,10,25,50,100,250,500,1000,2500,5000,10000}}`. Default buckets are `{0.5,1,5,10,25,50,100,250,500,1000,2500,5000,10000,30000,60000,300000,600000,1800000,3600000}`.
```

```suggestion
    description: Specifies the custom histogram buckets with a prefix matcher to separate the Istio mesh metrics from the Envoy stats, e.g. `{"istio":[1,5,10,50,100,500,1000,5000,10000],"envoy":[1,5,10,25,50,100,250,500,1000,2500,5000,10000]}`. Default buckets are `[0.5,1,5,10,25,50,100,250,500,1000,2500,5000,10000,30000,60000,300000,600000,1800000,3600000]`.
```

Current one is not valid json

---

can we specify what the unit is as well? milliseconds?

---

There's no unit. We have both byte count and milliseconds in use.